src/bigquery.ts

```diff
@@ -1104,7 +1104,7 @@ export class BigQuery extends common.Service {
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private static _getValue(value: any, type: ValueType): any {
-    if (value.type!) type = value;
+    if (value && value.type) type = value;
```

Comment:
If it's possible for a user to provide a null value for a custom type, the line below would try to access `value.value` and throw. I think we may want to short-circuit `null` values instead of letting the function continue to execute:

```js
if (value === null) {
  return null;
}
```

Reply:
Makes sense. Updated.